src/screens/view-book/view-book.component.js

```diff
@@ -7,33 +7,67 @@ import {
   Content,
   Text,
   Button,
-  View
+  View,
+  Icon,
+  ActionSheet,
+  Spinner
 } from 'native-base';
 
 import {
-  Image
+  Image,
+  Alert
 } from 'react-native';
 
 import styles from './view-book.styles';
 
 import GoBack from '../../components/go-back/go-back.component';
 import TaskList from '../../components/task-list/task-list.component';
 
+var BUTTONS = ['Sim', 'Não']
+var YES_INDEX = 1;
+var NO_INDEX = 2;
 export default class ViewBook extends React.Component {
+  constructor(props) {
+    super(props);
+    this.actionSheet = null;
+  }
+
+  getBookId() {
+    return this.props.book.id
+  }
 
   render() {
     return (
       <Container style={styles.container}>
         <View style={{ flex: 1 }}>
           <GoBack />
         </View>
-
-        <View style={{ flex: 1 }}>
-          <Text style={styles.title}>
-            {this.props.book.title}
-          </Text>
-        </View>
-
+        <Container style={{ flex: 1, flexDirection: 'row' }}>
+          <View style={{ flex: 1 }}>
+            <Text style={styles.title}>
+              {this.props.book.title}
+            </Text>
+          </View>
+          {this.props.user.id === this.props.book.userId ?
```

Comment:
Isso aqui nem é preciso eu acho, pq se o usuário não for dono do book deve ser barrado bem antes.